examples/workflows/script-annotations-inputs.yaml

```diff
@@ -44,7 +44,7 @@ spec:
       command:
       - python
       env:
-      - name: hera__script_annotations
```

Comment:
Why do we need to change the env var name?

Reply:
It’s not a necessary change, more of a stylistic choice @flaviuvadan mentioned we should probably fix